### PR TITLE
Use iOS Alert for help modal

### PR DESCRIPTION
**Pull request by fand**
## Changes

- Use iOS Alert for help modals
- Remove unused views

## Screenshot
alert

### CODE REVIEW

File: ZIGSIMPlus/Entities/ModalTexts.swift [hunk 1]
```diff
@@ -8,22 +8,23 @@
...
-    .proximity : "test9\nproximityです。",
-    .micLevel : "test10\nmicLevelです。",
-    .remoteControl : "test11\nremoteControlです。",
-    .ndi : "test12\nndiです。",
-    .nfc : "test13\nnfcです。",
-    .arkit : "test14\narkitです。",
-    .battery : "test16\nbatteryです。",
-    .applePencil : "test17\napplePencilです。"
+// Title and Body for help modal
+let modalTexts:[Command: (String, String)] = [
```
Comment (nozomuplus) [inline ZIGSIMPlus/Entities/ModalTexts.swift]: How about naming tuple, like
`(title: String, bodyText: String)` ?

**Comment by nozomuplus** [comment]
Good! Check my comment :smiley:

File: ZIGSIMPlus/Entities/ModalTexts.swift (hunk 1 above)
Comment (fand) [reply ZIGSIMPlus/Entities/ModalTexts.swift]: Whoa, I didn't know that feature.
Fixed in 5dedec7 🙏

**Comment by fand** [comment]
Thanks!! 😸

**Pull request merged**